### PR TITLE
Add ppc-only cases

### PR DESCRIPTION
**Pull request by chloerh**
- Assign address to panic device
- Passthrough non-available PCI device in guest

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

**Comment by iccaszhulili** [comment]
rh-caselink retest please

**Comment by chloerh** [comment]
Test result on power9
---

测试名称 | 运行时间 | 结果
-- | -- | --
panic_address | 28 秒 | Fixed
unavail_pci_device | 15 秒 | Passed

**Comment by kylazhang** [comment]
Please remove chinese in comments as some reviewers may not know what's the mean.

**Comment by chloerh** [comment]
@kylazhang Done

File: libvirt/tests/src/ppc_device.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            for i in range(5):
+                max_id += 1
+                # function must be <= 7
+                if max_id > 7:
+                    break
+                new_pci_id = '.'.join([prefix, str(max_id)])
+                new_pci_xml = libvirt.create_hostdev_xml(new_pci_id, xmlfile=False)
+                vmxml.add_device(new_pci_xml)
+            vmxml.sync()
+            logging.debug('Vm xml after adding unavailable pci devices:')
```
Comment (dzhengfy) [inline libvirt/tests/src/ppc_device.py]: Why not use one logging statement?

Comment (chloerh) [reply libvirt/tests/src/ppc_device.py]: You're right! Updated.

**Comment by chunfuwen** [comment]
@chloerh , suggest use more specific cfg file name:ppc_pci_device.cfg since there are large ranges of devices

**Comment by chloerh** [comment]
@chunfuwen , yes, I have considered that before. But this case file is not just about pci device, therefore the name of file will be either general or inaccurate.

**Pull request merged**